packages/firestore/src/api/timestamp.ts

```diff
@@ -64,10 +108,24 @@ export class Timestamp {
     }
   }
 
+  /**
+   * Convert a Timestamp to a JavaScript `Date` object. This conversion causes
```

Comment:
Should start with "Converts", not "Convert". Timestamp in backticks or make it lowercase like below.

Reply:
Fixed